dockertest/dockercmd_unittests.py

```diff
@@ -284,13 +282,13 @@ def test_no_fail_docker_cmd(self):
 
         docker_command = self.dockercmd.DockerCmd(self.fake_subtest,
                                                   'unittest_fail')
-        self.assertRaises(self.xceptions.DockerExecError,
+        self.assertRaises(self.output.xceptions.DockerExecError,
```

Comment:
Absolutely... but if you look at `dockertest/output_unittests.py` there are two already-existing instances of it (`self.output.xceptions.DockerOutputError`) dating to 2014-03-25 (57e6e60). That's where I got the idea.